src/app/app.component.spec.ts

```diff
@@ -20,6 +20,7 @@ import {
   Logger
 } from 'ngx-base';
 import { BsModalService } from 'ngx-bootstrap/modal';
+import { ModalModule } from 'ngx-bootstrap/modal';
```

Comment:
Why is this on a new line? It should be added to the import above.

Reply:
Because of auto import. I will fix this.